ui/app/src/components/PluginManagement/PluginManagement.tsx

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
Not sure what this is doing on this PR 🤔

Reply:
it should no be here